src/__experimental__/components/checkbox/checkbox.style.js

```diff
@@ -0,0 +1,185 @@
+import styled, { css } from 'styled-components';
+import PropTypes from 'prop-types';
+import baseTheme from '../../../style/themes/base';
+import FieldHelpStyle from '../field-help/field-help.style';
+import HiddenCheckboxStyle from './hidden-checkbox.style';
+import LabelStyle from '../label/label.style';
+import StyledHelp from '../help/help.style';
+import checkBoxClassicStyle from './checkbox-classic.style';
+
+const StyledCheckbox = styled.div`
+  ${({
+    checked, disabled, error, fieldHelpInline, inputWidth, reverse, size, theme
+  }) => css`
+    padding-top: 8px;
+
+    .carbon-checkbox__input {
+      padding-top: 1px;
+    }
+
+    svg {
+      background-color: ${theme.colors.white};
+      border: solid 1px ${theme.colors.border};
+    }
+
+    ${HiddenCheckboxStyle},
+    svg {
+      height: 16px;
+      position: absolute;
+      padding: 1px;
+    }
+
+    .carbon-checkbox__input,
```

Comment:
that should be a separate styled component

Reply:
Why? It's not being re-used anywhere else, is just a simple div, and adding a class to it gives the required outcome, without adding unnecessary layers of abstraction 🤔

Reply:
Following the discussion in Slack I've now made this change